It's done.

Determancer opened the pull request:
完事了，量略大，各位用空再看。

Determancer committed:
It's done.

Determancer committed:
Update zh_cn.lang

Determancer committed:
Update zh_cn.lang

Determancer committed:
Update zh_cn.lang

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+lore.flute=§5§o触摸、注视甚至思考这支长笛都会给我带来极其诡异的感受。那感觉仿佛是它并不存在于我的世界......然而它却在这里，在我的掌中。
+lore.saw_cleaver=§5§o这件武器...这一怪诞的奇异金属制品是工程学的杰作。它能够轻易地在其普通形态和延展形态间转换，同时也能使其使用者在战斗间灵活闪避，风格如同锯刃的远古主人。
+lore.wolf_medallion=§5§o这件金属饰品会在怪物靠近时发出低吼，向它的佩戴者发出警告。这无疑是远古之时猎人使用的物品。
+lore.bronze_sphere=§5§o不不不，这不可能，这绝不可能。我§l知道§5§o这个球体。我§l知道§5§o我曾经拿过它。我只是...没有记住它。到如今，它一定在我发现它的渔村中度过了漫长岁月吧？
```
如果这句话想要表达的在惊慌一点，或许“球体”可以改成“球”，改为“我认得这个球”；第二句中文里面很少这样表达，你可以写成“我好像拿起过它”，这里就不用“曾经”这个时态了，可以再压一压；“我只是...没能记住它”，“而岁月流转至今，在那个我发现它的小渔村里，想必它早已是历尽了沧海桑田吧？”

Determancer replied:
```suggestion
lore.bronze_sphere=§5§o不不不，这不可能，这绝不可能。我§l认得§5§o这个球。我§l知道§5§o我曾拿起它。我只是...没能想起它。时光流转至今，在我发现它的渔村里，想必它已历尽沧桑了吧？
```

TUsama replied:
我始终觉得“我§l知道§5§o我曾经拿过它”这句话非常的怪，因为“我曾经拿过它”这部分就已经暗含了“知道”“了解”，这也是为什么我们用这个词总是在后面加一个名词代词“我知道了”“我知道你的小把戏”，但是很少在后面跟一个句子成分完全的从句。我对此保留态度。

Determancer replied:
原文把知道一词专写为 KNOW，给这里标了重点，也就意味着你所说的暗含义其实是“我”想着重表达的内容，故有此译。“我拿起过”相当于回想当初，但这里其实是对“当初的回想”表示万分确信。

TUsama replied:
也是哦，这个就很难搞，因为这里严格上来说know后面跟的是一个宾语从句，但是在中文里可没有这种操作。“我清楚地记得我曾拿起它”这样可能比较舒服，换一个词。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+tooltip.held_villager.spinelessheartless=§5§o这个村民的脊骨和心脏都被摘除了。真是个无用的可怜人。
+tooltip.held_weeper.spineless=§5§o一个没有脊骨的泣者只不过是个头颅。
+tooltip.held_weeper.spineful=§5§o令人憎恶之物。
+tooltip.held_weeper.spinefulheartless=§5§o一个泣者无需心脏便可以哭泣。它会安然无恙。
```
一个？一名？

Determancer replied:
此处类同“一个教授”。另外泣者是一种生物。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+hit.idol.schiz5=停止。
+hit.idol.schiz6=立刻停止。
+hit.idol.schiz7=你真的该收手了。
+hit.idol.schiz8=§5停下
```
这里要不要加个逗号？总觉得可能会很鬼畜

Determancer replied:
这里的字会直接成为屏幕中央的大号浮现文字，直接贴脸。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+gui.city_mapper.incsize=提示：建议增加GUI的尺寸以便操作。
+
+gui.drowned.youdrowned=你溺水了
+gui.drowned.greatdreamer=相信伟大长梦者
```
Believe亦有信仰之意，此处？

Determancer replied:
此处是溺水了，**相信**谁能拯救自己。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+dagon.greeting.0=结盟非吾所求
+dagon.greeting.1=汝将为吾效劳
+dagon.greeting.2=成功当赐汝重赏
```
成则

Determancer replied:
我还是太菜了.jpg
```suggestion
dagon.greeting.2=成则赐汝重赏
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dagon.greeting.0=结盟非吾所求
+dagon.greeting.1=汝将为吾效劳
+dagon.greeting.2=成功当赐汝重赏
+dagon.greeting.3=失败后果无以言说
```
败则身堕无可名状之恐怖

Determancer replied:
过了
```suggestion
dagon.greeting.3=败果无以言说
```

WuzgXY-GitHub replied:
清秋绝了

TUsama replied:
败则尝无可名状之苦果？

Determancer replied:
> 败则尝无可名状之苦果？

保持神秘感，不言其苦，不述其细。
再者新修改方案刚好字数统一。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dagon.greeting.1=汝将为吾效劳
+dagon.greeting.2=成功当赐汝重赏
+dagon.greeting.3=失败后果无以言说
+dagon.greeting.4=汝为吾先驱
```
吾之？

Determancer replied:
```suggestion
dagon.greeting.4=汝为吾之先驱
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.stockpiler.greeting2=§5§o拜托，麻烦离我们§l远§5§o点。
+
+dweller.trustedstockpiler.greeting0=§5§o我以前错怪你了，外乡人。
+dweller.trustedstockpiler.greeting1=§5§o当然，我们现在把你视为朋友。
```
we deem you a friend now.

你现在就是我们的朋友了。

你还真把这个deem翻译成“视为”啊，这个词感觉得弱化。

Determancer replied:
```suggestion
dweller.trustedstockpiler.greeting1=§5§o当然，你现在是我们的朋友了。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.drunk.greeting35=......
+
+dweller.bartender.talk0={哼呣。}我们并不喜欢外乡人来这，但如果你真有需要，我倒不会拒绝让你留宿一晚。楼上还有一些床。
+dweller.bartender.talk1=不必在意那个身穿红衣的家伙。他是个嗜酒的蠢货，无休无止地从嘴里吐出谎言。
```
只会无休无止地从嘴里吐出谎言。

Determancer replied:
```suggestion
dweller.bartender.talk1=不必在意那个身穿红衣的家伙。他是个嗜酒的蠢货，只会无休无止地吐出谎言。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.bartendertrustedbar.talk0=嘿，如果你想休息的话，楼上还有些床。
+dweller.bartendertrustedbar.talk1=想喝杯酒？尽管开口就好。第一杯本店请客。
+dweller.bartendertrustedbar.talk1.option0=谢谢:那我笑纳了。
+dweller.bartendertrustedbar.talk1.option1=暂时不了，:谢谢。
```
虽然我知道原文就是如此，但是这个真的很怪。。。。。

Determancer replied:
我甚至在没学英语前口语就这么用 = =

WuzgXY-GitHub replied:
这是什么，分号？

Determancer replied:
这是对话时的功能符号，用于换行。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.bartendertrustedbar.talk1=想喝杯酒？尽管开口就好。第一杯本店请客。
+dweller.bartendertrustedbar.talk1.option0=谢谢:那我笑纳了。
+dweller.bartendertrustedbar.talk1.option1=暂时不了，:谢谢。
+dweller.bartenderrum.talk0=接好了，这是你能找到的最上品的朗姆酒。
```
> 接好了

来喽，

Determancer replied:
刚查了一下"There you go"，有“给你”的意思。假想一下酒馆的场景，“来，接好了，先生，这可是1995年的上好Java。”
所以我感觉问题不大。

TUsama replied:
There you go，我第一反应是电视剧里面那些小二的台词，所以直接是“来喽”，本土化一些，当然“接好了”未尝不可，取决于你决定是否如此本土化

TUsama reviewed (changes requested):
今晚的

Determancer commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dagon.greeting.3=失败后果无以言说
+dagon.greeting.4=汝为吾先驱
+dagon.greeting.5=致此金于渔民
+dagon.greeting.6=彼当知汝即吾使
```
```suggestion
dagon.greeting.6=彼当谓汝吾使
```

WuzgXY-GitHub replied:
可以解释一下这段的含义吗

Determancer replied:
他们应当称你为我的使者

WuzgXY-GitHub replied:
『这段』，这一整段文字在游戏架构、剧情发展中的位置，以及其本身想要叙述的内容。

正在 review，没法发消息；很好，就这么改吧。

Determancer replied:
> 『这段』，这一整段文字在游戏架构、剧情发展中的位置，以及其本身想要叙述的内容。

主角在得到大衮帮助后决定面见大衮，寻求结盟，但大衮要求主角臣服并为其效劳，随后命主角给渔民带去三块金子表明自己的使者身份，并让主角完成任务后与神像互动。

WuzgXY-GitHub commented:
爷来啦。iPad 还能使。

vassagolin commented:
同问加一




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "WuzgXY"<notifications@github.com&gt;; 
发送时间: 2020年8月17日(星期一) 上午8:39
收件人: "LWHK/LWHK-Simplified-Chinese-Translation"<LWHK-Simplified-Chinese-Translation@noreply.github.com&gt;; 
抄送: "永夜"<945822919@qq.com&gt;; "Assign"<assign@noreply.github.com&gt;; 
主题: Re: [LWHK/LWHK-Simplified-Chinese-Translation] It&#39;s done. (#61)





 
@WuzgXY-GitHub commented on this pull request.
 
 
In translation/beyondtheveil/zh_cn.lang:
 &gt; +building.clinic=诊所 +building.great_shrine=巨型神龛 + +angry.shoggoth.bad=一只沉浸于杀戮生灵的修格斯最终会将杀意转向它的主人。 + +################## +#大衮会话 +################## + +dagon.greeting.0=结盟非吾所求 +dagon.greeting.1=汝将为吾效劳 +dagon.greeting.2=成功当赐汝重赏 +dagon.greeting.3=失败后果无以言说 +dagon.greeting.4=汝为吾先驱 +dagon.greeting.5=致此金于渔民 +dagon.greeting.6=彼当知汝即吾使  
『这段』，这一整段文字在游戏架构、剧情发展中的位置，以及其本身想要叙述的内容。
 
—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub, or unsubscribe.

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+##################
+#物品（名称，描述和特殊效果）
```
```suggestion
#物品（名称、描述和特殊效果）
```
裂文唬客入门

Determancer replied:
绝了。但我不确定汉语的具体情况，据我所知逗号和顿号都能做分隔之用。

WuzgXY-GitHub replied:
入个门，随便的。

TUsama replied:
这是入门，绝了

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+item.surgeon_summons.name=外科医生传唤信
+
+
+lore.flute=§5§o触摸、注视甚至思考这支长笛都会给我带来极其诡异的感受。那感觉仿佛是它并不存在于我的世界......然而它却在这里，在我的掌中。
```
```suggestion
lore.flute=§5§o触摸、注视甚至思考这支长笛都会给我带来极其诡异的感受。那感觉就好像它并不存在于我的世界一样......然而它却确实在这里，就在我的掌中。
```
似乎破坏了美感。

Determancer replied:
原译问题不大。

WuzgXY-GitHub replied:
我感觉『仿佛』那段似乎有点不通顺

Determancer replied:
仿佛……=就像……一样
应该是个人主观感受所致，换个人就觉得通顺了。

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+
+lore.flute=§5§o触摸、注视甚至思考这支长笛都会给我带来极其诡异的感受。那感觉仿佛是它并不存在于我的世界......然而它却在这里，在我的掌中。
+lore.saw_cleaver=§5§o这件武器...这一怪诞的奇异金属制品是工程学的杰作。它能够轻易地在其普通形态和延展形态间转换，同时也能使其使用者在战斗间灵活闪避，风格如同锯刃的远古主人。
```
锯刃为什么会在这里

『风格』二字得改

Determancer replied:
如同已经说明了style相同，所以干脆去掉。
```suggestion
lore.saw_cleaver=§5§o这件武器...这一怪诞的奇异金属制品是工程学的杰作。它能够轻易地在其普通形态和延展形态间转换，同时也能使其使用者在战斗间灵活闪避，如同锯刃的远古主人。
```

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+lore.flute=§5§o触摸、注视甚至思考这支长笛都会给我带来极其诡异的感受。那感觉仿佛是它并不存在于我的世界......然而它却在这里，在我的掌中。
+lore.saw_cleaver=§5§o这件武器...这一怪诞的奇异金属制品是工程学的杰作。它能够轻易地在其普通形态和延展形态间转换，同时也能使其使用者在战斗间灵活闪避，风格如同锯刃的远古主人。
+lore.wolf_medallion=§5§o这件金属饰品会在怪物靠近时发出低吼，向它的佩戴者发出警告。这无疑是远古之时猎人使用的物品。
```
```suggestion
lore.wolf_medallion=§5§o这件金属饰品会在怪物靠近时发出低吼，向它的佩戴者发出警告以令其进入准备状态。毫无疑问，这是远古之时的猎人曾使用的物品。
```
或许略多余。

Determancer replied:
维持原译。

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+lore.saw_cleaver=§5§o这件武器...这一怪诞的奇异金属制品是工程学的杰作。它能够轻易地在其普通形态和延展形态间转换，同时也能使其使用者在战斗间灵活闪避，风格如同锯刃的远古主人。
+lore.wolf_medallion=§5§o这件金属饰品会在怪物靠近时发出低吼，向它的佩戴者发出警告。这无疑是远古之时猎人使用的物品。
+lore.bronze_sphere=§5§o不不不，这不可能，这绝不可能。我§l知道§5§o这个球体。我§l知道§5§o我曾经拿过它。我只是...没有记住它。到如今，它一定在我发现它的渔村中度过了漫长岁月吧？
+lore.crucible=§5§o往昔的斩魔人所挥舞的武器。极其强大，但是每次使用后需要很长的再充能时间。
```
```suggestion
lore.crucible=§5§o往昔的斩魔人所挥舞的武器。极其强大，但是每次使用后都需要再充能很长的一段时间。
```

Determancer replied:
```suggestion
lore.crucible=§5§o往昔的斩魔人所挥舞的武器。极其强大，但是每次使用后需要再充能很久。
```

WuzgXY-GitHub replied:
```suggestion
lore.crucible=§5§o往昔的斩魔人所挥舞的武器。极其强大，但是每次使用后都需要再充能很久。
```

Determancer commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.DEEPROAR.title=怒吼
+research.DEEPROAR.stage.0=在深潜化时，我无法使用饰品的主动效果，也无法打开饰品能力菜单。<BR>然而，在这一形态下，我可以发出令人血液凝固的怒吼，让我附近的所有实体陷入莫大的恐怖之中，通过按下饰品使用主动效果的同一按键即可释放该技能。<BR>将恐惧刺入我的敌人的心脏，即是比任何时刻都更接近神。
+
+research.TONEWDEPTHS.title=深渊之渊
```
```suggestion
research.TONEWDEPTHS.title=极渊之渊
```

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+use.coral_staff.clear=清除最后一次保存的位置。
+use.blood_sigil.amount=这个印记现在连接着 %1$s 个亡灵生物。
+use.blood_sigil.emptysigil=这个印记是空的。
+use.blood_sigil.emptywell=这个血井中无更多该种类的亡灵生物。
```
```suggestion
use.blood_sigil.emptywell=这个血井中并无更多该种类的亡灵生物。
```
```suggestion
use.blood_sigil.emptywell=这个血井中没有更多该种类的亡灵生物。
```

Determancer replied:
采用第二个。
```suggestion
use.blood_sigil.emptywell=这个血井中没有更多该种类的亡灵生物。
```

WuzgXY-GitHub reviewed (approved):
第一波

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperend.talk5=你来自什么世界是无关紧要的，因为真相遍布一切领域。|:因为真相深藏于帷幕彼端，栖居在一切宇宙之间的区域。
+dweller.lhkeeperend2.talk0=继续做你要做的事吧，杀死我。我的死无足轻重。||:你的死亡亦是如此。
+
+dweller.scholar.talk0=哦天哪！一位旅行者！|:您好！嗨！向您致敬！热烈欢迎！诚挚问候！
```
这个真的很不像是口语。

Determancer replied:
这是位学者，说话时书面语和口语夹杂，因此此处Worded as Intended

WuzgXY-GitHub replied:
我觉得很合适。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperend.talk2=另一个真相。我是微不足道的。:在这个世界上还有不计其数的人和我完全一样。精密的复制品。:我不过是因果的一支，无关紧要，以致你可以毫不犹豫地为我送来死亡。
+dweller.lhkeeperend.talk3=我有一个问题，外乡人。:你真的拥有一个具备无穷复杂性的心智吗？我很抱歉这么说，但是你不可能有。亿万突触何其精妙，然而，你的心智也迟早会被复制。就像我的心智一样。
+dweller.lhkeeperend.talk4=这便是真相的片断，外乡人，这是专门为你准备的。你的生命一文不值。
+dweller.lhkeeperend.talk5=你来自什么世界是无关紧要的，因为真相遍布一切领域。|:因为真相深藏于帷幕彼端，栖居在一切宇宙之间的区域。
```
我更倾向于“存在于一切领域”，哲学病。

“遍布于”也可以，感觉少个介词。

Determancer replied:
“通信网络遍布全国各地。”

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.craftedwatery_cradle.text=合成三个沉梦水篮。
+
+research.SPINES.title=拔出脊骨
+research.SPINES.stage.0=凡人对神而言不过是蝼蚁。<BR>拔出他们的脊骨，让他们爬。
```
？

Determancer replied:
我斟酌了一番，发现让他们爬语气不错。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.IdolInteract.text=在渔村的神像前祈祷。
+
+research.SLUGS.title=蛞蝓
+research.SLUGS.stage.0=与伟大长梦者共鸣是至高的荣耀。实不相瞒，给祂留下印象并使祂对我产生兴趣，无可置疑地是我有史以来最大的成就。<BR>然而共鸣的强度十分微弱，因为我太过弱小，而我的存在无可救药地与主世界牢牢绑定。<BR>祂深知如此。祂希望帮助我。真的，祂是这么想的！<BR>这些生物就如同它们降生时所在的水一样，黏滑、无定形、难以捕捉。它们就是一些渔村居民赖以维生的白色蛞蝓。它们是共鸣之门的钥匙。<BR>我需要一个诱捕器，而伟大长梦者赐予了我制造它的知识。具有讽刺意味的同时又充满意义的是，那些无定形的、近乎混沌的蛞蝓却会被形态完美而有序的食盐所吸引！<BR>蛞蝓诱捕器必须被放在水中。其上方不能有水，但其四个侧面必须有水。为了发挥最大性能，它应被放置在11×11的方形水域的中央。<BR>如果达到要求，我就能看到微小的颗粒，而不久后我就能看到水面上出现蛞蝓充满水的身体 ...
```
> 具有讽刺意味的同时又充满意义的是

多么讽刺，却又耐人寻味，那些无定形的、近乎混沌的蛞蝓却会被形态完美而有序的食盐所吸引！

Determancer replied:
```suggestion
research.SLUGS.stage.0=与伟大长梦者共鸣是至高的荣耀。实不相瞒，给祂留下印象并使祂对我产生兴趣，无可置疑地是我有史以来最大的成就。<BR>然而共鸣的强度十分微弱，因为我太过弱小，而我的存在无可救药地与主世界牢牢绑定。<BR>祂深知如此。祂希望帮助我。真的，祂是这么想的！<BR>这些生物就如同它们降生时所在的水一样，黏滑、无定形、难以捕捉。它们就是一些渔村居民赖以维生的白色蛞蝓。它们是共鸣之门的钥匙。<BR>我需要一个诱捕器，而伟大长梦者赐予了我制造它的知识。多么讽刺却又耐人寻味的是，那些无定形的、近乎混沌的蛞蝓竟会被形态完美而有序的食盐所吸引！<BR>蛞蝓诱捕器必须被放在水中。其上方不能有水，但其四个侧面必须有水。为了发挥最大性能，它应被放置在11×11的方形水域的中央。<BR>如果达到要求，我就能看到微小的颗粒，而不久后我就能看到水面上出现蛞蝓充满水的身体。它们就好像是凭空出现的，并且在片刻后就会突然消失。我必须将捕蛞蝓器小心的对准它们，然后再使用。<BR>再然后...我必须吃下它们。这是唯一的办法。我并不喜欢，但我必须这么做。<BR>一旦我吃下一些蛞蝓，比如说，大约20只，我就应当再次与神像接触。
```

TUsama replied:
如果你决定用我的版本，但是后面又加上了一个“的是”，那中间的逗号可以删掉。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+research.CANOE.title=深海之赠
+research.CANOE.stage.0=旧渔村居民的日常饮食中除了蛞蝓，主要是鱼。他们的潮木桶中装满了鳕鱼。<BR>他们好像是使用独木舟去捕鱼。这些独木舟由木匠制造，木匠是身穿深灰色服装的居民，住在内有寻常橡木梯子的双层房屋之中。<BR>并非每一个渔村都有木匠，但他们相当常见。
+research.CANOE.stage.1=据木匠称，渔民们会乘着独木舟入海...然后鱼会自己主动找上他们？<BR>我也应该试试，但不应该在随意的水域进行，而应该到大海中尝试一下。<BR>如果确实发生了什么事情，我应该多保持一段时间，然后以吃下相当数量的蛞蝓作为结束。在那之后，我应该去共鸣。
```
我应该“让这件事”多保持一段时间

Determancer replied:
这里指的是在船里多呆一会儿，发生的事情是转瞬即逝的。
```suggestion
research.CANOE.stage.1=据木匠称，渔民们会乘着独木舟入海...然后鱼会自己主动找上他们？<BR>我也应该试试，但不应该在随意的水域进行，而应该到大海中尝试一下。<BR>如果确实发生了什么事情，我应该在那里多待一段时间，然后以吃下相当数量的蛞蝓作为结束。在那之后，我应该去共鸣。
```

TUsama replied:
所以具体是怎么样的，我还以为是什么刮风闪电，然后忍久一点之类的。

Determancer replied:
具体是一条鱼会蹦上来。

TUsama replied:
。。。。。。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.DREAMBOTTLE.title=饮梦瓶
+research.DREAMBOTTLE.stage.0=我在受洗时流了一滴眼泪。<BR>那是真相的点滴，提醒我牢记自己毫无价值。<BR>但是，正是那滴眼泪引导我*认知*。<BR>我无法大量的收集自己的眼泪，毕竟我不是熟练的泣者，对吧？<BR>我不是吧？<BR>我将向除我自身之外的其他躯体寻求帮助。
+research.DREAMBOTTLE.stage.1=通过泣者和泣囊，我被授以知识，通过梦境，我将放大并利用这些知识。<BR>饮梦瓶正是我需要合成的物品。这是一种无价宝具，能使我每天无限次地进入梦境，并且在一场梦中同时浸入多个记忆。<BR>唯一的代价是需要泪水。但并非我自己的泪水。
+research.DREAMBOTTLE.stage.2=饮梦瓶可储存泪水。对着一个泪瓶使用即可将饮梦瓶填至最大容量，再次使用即可将饮梦瓶内的泪水尽可能多的存入泪瓶之中。<BR>饮梦瓶至多能存储4桶泪。<BR>接下来在未潜行时右击使用饮梦瓶即可在其中放入记忆安瓿，至多能放置四个。<BR>现在，无论何时，只要我在潜行时喝下饮梦瓶的内容物，我就能如己所愿，随时随地入梦。<BR>每个梦都会消耗一桶泪，这即是兑换的比率。<BR>随着我了解这一全新的器具，梦之艺术为我展现出新的可能。
```
接下来，在未潜行时

Determancer replied:
```suggestion
research.DREAMBOTTLE.stage.2=饮梦瓶可储存泪水。对着一个泪瓶使用即可将饮梦瓶填至最大容量，再次使用即可将饮梦瓶内的泪水尽可能多的存入泪瓶之中。<BR>饮梦瓶至多能存储4桶泪。<BR>接下来，在未潜行时右击使用饮梦瓶，即可在其中放入记忆安瓿，至多能存放四个。<BR>现在，无论何时，只要我在潜行时喝下饮梦瓶的内容物，我就能如己所愿，随时随地入梦。<BR>每个梦都会消耗一桶泪，这即是兑换的比率。<BR>随着我了解这一全新的器具，梦之艺术为我展现出新的可能。
```

TUsama committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: WuzgXY <62000315+WuzgXY-GitHub@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: WuzgXY <62000315+WuzgXY-GitHub@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+dweller.lhkeeper.talk0=噢。|{一个旅行者}[...]:欢迎。
+dweller.lhkeeper.talk1=我猜你是在永无止尽的旅途中，偶然发现了我们的渔村。:我只能对你表示羡慕，毕竟我只能被困在这里，{看守这座灯塔。
+dweller.lhkeeper.talk2=虽然如此，但我有个疑惑，像你这样的探险家如何看待我们这卑微的渔村？|这[所有]你肯定欣赏的美景，给你留下的最深的印象是什么？
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

“humble”译为“卑微”正确？我知道你想说的是这个村庄与阿撒托斯之间的地位对比，但是故事进展到这里，冒险者是否已经知道了真相？我个人还是觉得可能要翻译为“简陋比较妥当。
```suggestion
dweller.lhkeeper.talk2=虽然如此，但我有个疑惑，像你这样的探险家如何看待我们这卑微的渔村？|这[所有]你肯定欣赏的美景，给你留下的最深的印象是什么？
```
虽然如此，但我有个疑惑，像你这样的探险家如何看待我们这简陋的渔村？|这[所有]你肯定欣赏的美景，给你留下的最深的印象是什么？

Determancer replied:
```suggestion
dweller.lhkeeper.talk2=虽然如此，但我有个疑惑，像你这样的探险家如何看待我们这简陋的渔村？|这[所有]你肯定欣赏的美景，给你留下的最深的印象是什么？
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperlackknowledge.talk1=我很怀疑。你的心智尚未开化。
+dweller.lhkeeperhasknowledge.talk0=我...我确实能感受到你的内心存在一种{渴望...}:一种对知识的贪欲。|一种好奇心，一种愚蠢但{美妙的品质。
+dweller.lhkeeperlecture.talk0=你想了解更多我们的历史吗？了解我们经受的所有{折磨}？
+dweller.lhkeeperlecture.talk1=你是否想了解更多你祖先对我们所做的事，而这些事仅仅出于对[未知]的恐惧？
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
dweller.lhkeeperlecture.talk1=你是否想了解更多你祖先对我们所做的事，而这些事仅仅出于他们对[未知]的恐惧？
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperhasknowledge.talk0=我...我确实能感受到你的内心存在一种{渴望...}:一种对知识的贪欲。|一种好奇心，一种愚蠢但{美妙的品质。
+dweller.lhkeeperlecture.talk0=你想了解更多我们的历史吗？了解我们经受的所有{折磨}？
+dweller.lhkeeperlecture.talk1=你是否想了解更多你祖先对我们所做的事，而这些事仅仅出于对[未知]的恐惧？
+dweller.lhkeeperlecture.talk1.option0=告诉我更多。:我想了解。
```
|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
dweller.lhkeeperlecture.talk1.option0=请告诉我更多。:我想了解。
```
祈使句

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperlecture.talk0=你想了解更多我们的历史吗？了解我们经受的所有{折磨}？
+dweller.lhkeeperlecture.talk1=你是否想了解更多你祖先对我们所做的事，而这些事仅仅出于对[未知]的恐惧？
+dweller.lhkeeperlecture.talk1.option0=告诉我更多。:我想了解。
+dweller.lhkeeperlecture.tellmetalk0=看到你能追寻而非逃避知识，实在是令我耳目一新。|:尽管，当然了...|我必须警告你。|{:知识即恐惧。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
dweller.lhkeeperlecture.tellmetalk0=看到你能追寻而非逃避知识，实在是令我眼前一亮。|:尽管，当然了...|我必须警告你。|{:知识即恐惧。
```
耳目一新强调的是事物给人带来的形象新鲜感受，眼前一亮不仅强调了新鲜感受，还强调了心中的诧异，我个人觉得是后者比较好。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperlecture2.lecturetalk2=然而这里的地面过去并非如此。:与一些人所想的不同，我们并不是因为被这里的土壤所吸引而定居于此。:事实上，我们恰恰是地面变成这番样貌的原因。
+dweller.lhkeeperlecture2.lecturetalk3=我们对神祇的崇拜使之发生，而如果这无法证明我们的神真实存在，那么我不知道还有什么证明。:可惜的是，[你祖先]中的许多人并不喜欢这一想法，不相信有神比他们所信仰的神灵更加真实。
+dweller.lhkeeperlecture2.lecturetalk4=实际上他们万分厌恶这一想法，憎恶能将地面变得漆黑潮湿的神祇，于是他们视我们为“异教徒”，然后开始猎杀我们。[一个接着一个。]
+dweller.lhkeeperlecture2.lecturetalk5={我们的家园被付之一炬，我们的人民血流成河。}我们的父辈，那些侥幸存活者，失去了一切。这是一场{痛绝人寰的惨剧。}{这是一次[大屠杀]。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
dweller.lhkeeperlecture2.lecturetalk5={我们的家园被付之一炬，我们的人民血流成河。}我们的父辈，那些侥幸存活者，失去了一切。这是一场{惨绝人寰的悲剧。}{这是一次[大屠杀]。
```
痛绝人寰又是smjb

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperlecture2.lecturetalk9.option1=我理解了。:我向你表示同情。
+dweller.lhkeeperlecture2.friendslecturetalk0=那与你无关，因为你依然是个外乡人。:但事态可以有所变化。
+dweller.lhkeeperlecture2.thankstalk0=而我向你表示感激。因为你聆听了我的故事，我们的历史。:因为你使我从过往知识的重担中脱离。
+dweller.lhkeepergratitude.talk0=你还想知道更多的话，尽管吩咐。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
dweller.lhkeepergratitude.talk0=你还想知道更多的话，就问吧。
```
吩咐是不是有一点过于。。。小二了

Determancer replied:
```suggestion
dweller.lhkeepergratitude.talk0=你还想知道更多的话，尽管问我就好。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperpast.talk0=这无关紧要。过去的就让它留在过去吧，我们并非能穿越时光的人。只论当下。
+dweller.lhkeeperend.talk0=允许我留下几句遗言？|万分感谢，旅行者。:我会尽快说完。
+dweller.lhkeeperend.talk1=让我们从一个客观真相开始说起。一个数理层面的事实。:你所能想到的最大数字，与无穷大相比也接近零。
+dweller.lhkeeperend.talk2=另一个真相。我是微不足道的。:在这个世界上还有不计其数的人和我完全一样。精密的复制品。:我不过是因果的一支，无关紧要，以致你可以毫不犹豫地为我送来死亡。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
dweller.lhkeeperend.talk2=另一个真相。我是微不足道的。:在这个世界上还有不计其数的人和我完全一样。都是精密的复制品。:我不过是众多因缘之和合罢了，无关紧要，以致你可以毫不犹豫地为我送来死亡。
```

Determancer replied:
```suggestion
dweller.lhkeeperend.talk2=另一个真相。我是微不足道的。:在这个世界上还有不计其数的人和我完全一样。都是精密的复制品。:我不过是一系列因果的集合，无关紧要，以致你可以毫不犹豫地为我送来死亡。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.lhkeeperend.talk0=允许我留下几句遗言？|万分感谢，旅行者。:我会尽快说完。
+dweller.lhkeeperend.talk1=让我们从一个客观真相开始说起。一个数理层面的事实。:你所能想到的最大数字，与无穷大相比也接近零。
+dweller.lhkeeperend.talk2=另一个真相。我是微不足道的。:在这个世界上还有不计其数的人和我完全一样。精密的复制品。:我不过是因果的一支，无关紧要，以致你可以毫不犹豫地为我送来死亡。
+dweller.lhkeeperend.talk3=我有一个问题，外乡人。:你真的拥有一个具备无穷复杂性的心智吗？我很抱歉这么说，但是你不可能有。亿万突触何其精妙，然而，你的心智也迟早会被复制。就像我的心智一样。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

我有点犹豫。。。这里谈到的是“突触”，生物学上的概念，后面的那个“mind“是否也是一个生物学概念？因为如果是”心智“，突触这一概念放在这里就很难理解了。我觉得这里说的是：即便你的大脑如此的复杂，但是依然会被复制。

Determancer replied:
我觉得问题不大。心智也是由突触产生的。

Determancer replied:
```suggestion
dweller.lhkeeperend.talk3=我有一个问题，外乡人。:你真的拥有一个具备无穷复杂性的心智吗？我很抱歉这么说，但是你不可能有。亿万突触何其精妙，然而，你的头脑也迟早会被复制。就像我的头脑一样。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+dweller.scholarshoggoth.sorrytalk2.option0=什么声音？
+dweller.scholarshoggoth.sorrytalk2.option1=是的。
+dweller.scholarshoggoth.enlightentalk0=哦不，我不打算告诉你。:但汝实为幸运，我的好伙计。
+dweller.scholarshoggoth.enlightentalk1=然而我担心汝脑袋里的声音迟早会告诉汝何以为之一。
```
|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

我从未见过”何以为之一“这一表达，我宁愿用稍微白话一点的表达。

Determancer replied:
原本是Work as Intended，因为这个学者文白夹杂。不过确实略怪。
```suggestion
dweller.scholarshoggoth.enlightentalk1=然而我担心汝脑袋里的声音迟早会告诉汝如何制作它。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+research.FUMESPREADER.title=记忆
+research.firstMemory.text=制作唤知之忆。
+research.FUMESPREADER.stage.0=一段模糊的新记忆从我上次的梦境中浮现。<BR>一段关于唤醒知识的记忆，但它如此模糊，很可能并不属于我。<BR>不过我应该尽可能地发掘这段记忆中的内容。话说回来，日常经历的所留下的记忆能对梦境有多大的影响？<BR>啊，但我知道一些鲜为人知的事情。记忆可以被装入瓶中。这需要通过唤忆池来完成。<BR>唤忆池是一种盛水的石制水池，我可以在水池上方放置能够唤起记忆的物品，然后再用记忆安瓿来封装记忆。<BR>就唤知之忆而言，恶魂之泪正是能够唤起记忆的物品，虽然我完全看不出这其中的关联。<BR>作为旁注，如果这份知识并非来自记忆，那么它又是从何处突然到来？哦，多么讽刺！
```
|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
research.FUMESPREADER.stage.0=一段模糊的新记忆从我上次的梦境中浮现。<BR>一段关于唤醒知识的记忆，但它如此模糊，很可能并不属于我。<BR>不过我应该尽可能地发掘这段记忆中的内容。话说回来，日常经历的所留下的记忆能对梦境产生影响吗？<BR>啊，尽管我不知道这一问题的回答，但我却知道一些鲜为人知的事情。记忆可以被装入瓶中。这需要通过唤忆池来完成。<BR>唤忆池是一种盛水的石制水池，我可以在水池上方放置能够唤起记忆的物品，然后再用记忆安瓿来封装记忆。<BR>就唤知之忆而言，恶魂之泪正是能够唤起记忆的物品，虽然我完全看不出这其中的关联。<BR>作为旁注，如果这份知识并非来自记忆，那么它又是从何处突然到来？哦，多么讽刺！
```
漏译这个转折关系，原文是”Ah, but I know something very few others do.“，这个”other“你或许可以以一种比我这个更压缩的形式编进去，我就想不细想了

Determancer replied:
这里强调的是产生影响极小，然后用封装的方式来增强。
```suggestion
research.FUMESPREADER.stage.0=一段模糊的新记忆从我上次的梦境中浮现。<BR>一段关于唤醒知识的记忆，但它如此模糊，很可能并不属于我。<BR>不过我应该尽可能地发掘这段记忆中的内容。话说回来，日常经历的所留下的记忆又能对梦境有多大的影响呢？<BR>啊，虽然我无法确定这一点，但我知道一些鲜为人知的事情。记忆可以被装入瓶中。这需要通过唤忆池来完成。<BR>唤忆池是一种盛水的石制水池，我可以在水池上方放置能够唤起记忆的物品，然后再用记忆安瓿来封装记忆。<BR>就唤知之忆而言，恶魂之泪正是能够唤起记忆的物品，虽然我完全看不出这其中的关联。<BR>作为旁注，如果这份知识并非来自记忆，那么它又是从何处突然到来？哦，多么讽刺！
```

TUsama replied:
> can Dreams be influenced from the Memories of everyday experiences?

我没看到对于程度的描写，只看到有无的描写。

Determancer replied:
> > can Dreams be influenced from the Memories of everyday experiences?
> 
> 我没看到对于程度的描写，只看到有无的描写。

联系一下上下文：如果日常生活的记忆足用，那边无需使用记忆安瓿。所以“我”才会在困惑于日常记忆实际效果时说**但**“我”知道一个秘密。这里的目的应该是要放大效果。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.metalDream.text=梦见金铁。
+
+research.SLEEPCHAMBER.title=沉眠室
+research.SLEEPCHAMBER.stage.0=我一直为自己的能力感到自豪，它使得我能按己所需而保持清醒，持续工作和随意走动，把睡眠的时间留给更重要的事情。<BR>噢，可极具讽刺意味的是，如今我万分需要睡眠，睡多久都不够。<BR>我必须入梦，持续地入梦，周而复始。一日一次，且只能夜间入梦，这根本不够，完全不够。我必须集中精力去制造某种装置，以使我摆脱那种限制，至少能在某种程度上摆脱它。<BR>首先，我需要一个尺寸合适的合成站。齿轮工作台将能帮助我制造更多的复杂物体。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
research.SLEEPCHAMBER.stage.0=只要我想，我就可以一直保持清醒，任意工作、走动，我对此颇为自豪。因为这可以把睡眠的时间腾给更重要的事情。<BR>噢，可极具讽刺意味的是，如今我万分需要睡眠，睡多久都不够。<BR>我必须入梦，持续地入梦，周而复始。而入梦只能一日一次，且只能夜间入梦，这根本不够，完全不够。我必须集中精力去制造某种装置，以使我摆脱那种限制，至少能在某种程度上摆脱它。<BR>首先，我需要一个尺寸合适的合成站。齿轮工作台将能帮助我制造更多的复杂物体。
```
我第一遍看我甚至没理解想说什么。。

Determancer replied:
```suggestion
research.SLEEPCHAMBER.stage.0=只要我想，我就可以一直保持清醒，随意工作、走动，我对此颇为自豪，因为这样一来，我就可以把睡眠的时间腾给更重要的事情。<BR>噢，可极具讽刺意味的是，如今我万分需要睡眠，睡多久都不够。<BR>我必须入梦，持续地入梦，周而复始。如今入梦仅能做到一日一次，且必须在夜间进行，这根本不够，完全不够。我必须集中精力去制造某种装置，以使我摆脱这一限制，至少能在某种程度上摆脱它。<BR>首先，我需要一个尺寸合适的合成站。齿轮工作台将能帮助我制造更多的复杂物体。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+research.SLEEPCHAMBER.title=沉眠室
+research.SLEEPCHAMBER.stage.0=我一直为自己的能力感到自豪，它使得我能按己所需而保持清醒，持续工作和随意走动，把睡眠的时间留给更重要的事情。<BR>噢，可极具讽刺意味的是，如今我万分需要睡眠，睡多久都不够。<BR>我必须入梦，持续地入梦，周而复始。一日一次，且只能夜间入梦，这根本不够，完全不够。我必须集中精力去制造某种装置，以使我摆脱那种限制，至少能在某种程度上摆脱它。<BR>首先，我需要一个尺寸合适的合成站。齿轮工作台将能帮助我制造更多的复杂物体。
+research.SLEEPCHAMBER.stage.1=这台被我称为沉眠室的机器非常奇妙。建造它所需的机械学、生物学和化学的知识，远超我在知识界的寻常视野，这使我不得不将其视为......异星的技术？<BR>然而它就这么来到了我的手中，如此突然，而且并非通过试验得来，而仿佛是它自己寻来。<BR>噢，我还是另找时间研究这神秘现象吧。<BR>使用沉眠室时首先要将其打开，我进入其中之后必须关上门以开始入梦。它能使我每天额外入睡一次，并且在梦后立刻醒来，时间在此期间不会流逝。但这瞬间足以让梦游离我身，并依照旁边熏香器的指示发挥作用，使用方法就像普通的床一样。<BR>这件发明令人难以置信，但我感到我仅仅触及了表面，这场旅途能指引我走向更深处。<BR>这不过是一块垫脚石，借此我将粉碎一切我自以为知的虚假真相。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
research.SLEEPCHAMBER.stage.1=这台被我称为沉眠室的机器非常奇妙。建造它所需的机械学、生物学和化学的知识，远超我那平庸的知识范围，这使我不得不将其视为......异星的技术？<BR>然而它就这么来到了我的手中，如此突然，而且并非通过试验得来，而仿佛是它自己寻来。<BR>噢，我还是另找时间研究这神秘现象吧。<BR>使用沉眠室时，首先要将其打开，我进入其中之后必须关上门以开始入梦。它能使我每天额外入睡一次，并且在梦后立刻醒来，时间在此期间不会流逝。但这瞬间足以让梦游离我身，并依照旁边熏香器的指示发挥作用，使用方法就像普通的床一样。<BR>这件发明令人难以置信，但我感到我仅仅触及了表面，这场旅途能指引我走向更深处。<BR>这不过是一块垫脚石，借此我将粉碎一切我自以为知的虚假真相。
```

TUsama reviewed (changes requested):
这也太多了

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Clefal <1716976420@qq.com>

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Clefal <1716976420@qq.com>

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Clefal <1716976420@qq.com>

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Clefal <1716976420@qq.com>

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Clefal <1716976420@qq.com>

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.EFFECTDREAMS.stage.0=如今，我的梦境已经能够授予我知识，这些知识上至奥术的秘密和奇迹，下至如我脚下地面之类的简单内容。<BR>然而，我相信......§l不§r！我完全§l确信§r还有更加深奥的内容。<BR>可能我还无法使自己的梦境影响周围的世界，但我不禁想到，应该有种办法来让梦境影响到...我自身？并非我的意识，而是我的躯体...<BR>听起来好像绝无可能，但我已经越过了现实与梦境的界限，难道不是吗？
+research.EFFECTDREAMS.stage.1=一段记忆影响梦境的方式高深莫测。它们是讯息，这无可置疑，但我该如何理解讯息的真实含义，考虑到这讯息的来源如此诡谲、怪异，就像是...梦境一般？<BR>我将记录那些记忆，能够改变我身上已有的状态效果的记忆。
+research.effectDream.text=唤起能够触动我自身的记忆。
+research.EFFECTDREAMS.addenda.power=一段关于力量的记忆，在我击杀体型较大的野兽时显现。那是个值得缅怀的时刻，令人满足。<BR>通过它，我的梦将能增加身上效果的强度。<BR>正面效果和负面效果都会被增强。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

~~我倾向于认为这里的as是“当...时”的意思，指的是记忆的内容，但是你这里翻译成在....显现，就比较奇怪，因为你这样子翻译，记忆本身的内容是没有说明的，只是在一些特殊的时刻显现罢了。
我个人认为，如果翻译为“当...时”，实际上就是一个类似于状语从句的结构，但是这里实际上指明的是记忆在时间上的内容，~~

~~为什么动词是一般现在时？~~

https://wenku.baidu.com/view/cd9f0a78b207e87101f69e3143323968011cf4ad.html
我觉得，是不是方式状语从句？

Determancer replied:
这个模组中的记忆（memory）一律被写成了Memory，以表明它与普通的记忆有所不同。许多记忆并非来自“我”自身，而是在特定的情形下从未知的地方进入了“我”的意识。最典型的例子就是Memory of Learning。
我是根据这个思路此类语段的。

TUsama replied:
我知道你是这个思路，但是按照这个思路，从句的动词不应该是一般过去式吗？你只有有过这个普通的记忆，你才能以这个记忆作为触发条件。但是这里的动词全部都是一般现在时，这是令我最疑惑的点。

WuzgXY-GitHub replied:
👀

Determancer replied:
as这个我再斟酌一下，想个好的方案。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.EFFECTDREAMS.stage.1=一段记忆影响梦境的方式高深莫测。它们是讯息，这无可置疑，但我该如何理解讯息的真实含义，考虑到这讯息的来源如此诡谲、怪异，就像是...梦境一般？<BR>我将记录那些记忆，能够改变我身上已有的状态效果的记忆。
+research.effectDream.text=唤起能够触动我自身的记忆。
+research.EFFECTDREAMS.addenda.power=一段关于力量的记忆，在我击杀体型较大的野兽时显现。那是个值得缅怀的时刻，令人满足。<BR>通过它，我的梦将能增加身上效果的强度。<BR>正面效果和负面效果都会被增强。
+research.EFFECTDREAMS.addenda.stillness=一段关于静止的记忆，在我一次又一次地被困在沉眠室中的期间到来。<BR>我身上已有的效果，我绝不让它们离开。通过这一记忆，我的梦将能增长效果的时间。所有效果都是如此。<BR>它们逃不掉。一个都别想逃。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

“imprison”翻译成“困住”有点不妥，因为你是自愿进入沉眠室，你不能说你自愿进入一个地方，又说它困住了你。这里强调的应该是那种与世隔绝的感觉，因为待在沉眠室里是感觉不到时间流动的。
这里的“as”和上面一样。

Determancer replied:
```suggestion
research.EFFECTDREAMS.addenda.stillness=一段关于静止的记忆，在我一次又一次地在沉眠室里与世隔绝的期间到来。<BR>我身上已有的效果，我绝不让它们离开。通过这一记忆，我的梦将能增长效果的时间。所有效果都是如此。<BR>它们逃不掉。一个都别想逃。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+research.HUMANDREAMS.title=他者之梦
+research.HUMANDREAMS.stage.0=有时候，当我入梦之时，我会有一种怪异的感受。<BR>像我一样的人开始出现在我的思想之中，其中有些人我甚至从未见过。然而，我隐隐感到它不仅仅是我的意识和潜意识凭空构造的产物。<BR>没错。我认为我们的梦会聚合到一点，人们的梦境于此互触，而出现在我思想之中的那些人都是§l真实§r的。<BR>我坚信着，凭借我的“入梦专业”，我能够对其他人有更多的了解，甚至在他们一无所知的情况下探查他们的梦境。<BR>我应该好好利用这一能力。我可以试着生成一段新的记忆，这段记忆以社会交往的经历为核心，如此一来便能将我的梦境指向他人的梦境。<BR>或许尝试和村民发起一次简单的交易会有所帮助？
+research.HUMANDREAMS.stage.1=我有理由相信，在其怪异的、非维度态的存在位面中，诸梦彼此相邻。每一个梦都邻接余众。在同一时刻，在任何时间。<BR>虽然它们在世界中的位置可能会变换（例如在它们直线坠落穿过基岩之时），我的梦仍然与他者的相邻。<BR>这是突破性的发现。作为一个比大多数人都更熟练的梦者，我能够用自己的梦去探查他人的梦，无视他们自以为拥有的隐私，增进对他们的了解。我应该继续探索这一能力，以便开发其全部潜能。<BR>然而，这一新的知识给我带来了新的恐惧。<BR>这种恐惧来自一个想法：或许有其他的梦者，比我更为强大，而我对梦境过度的入侵只会使我变得更加脆弱。<BR>其他的梦者，他们甚至可能不是人类。<PAGE>关于人类的记忆能通过与哪些愚昧村民中的一员发起交易来唤起。他们迟早会为自己愚蠢的定价付出代价。<BR>这段记忆现在所能引发的梦境并不会带来很大影响。它 ...
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
research.HUMANDREAMS.stage.1=我有理由相信，在其怪异的、非维度态的存在位面中，诸梦彼此相邻。每一个梦都邻接余众。在同一时刻，在任何时间。<BR>虽然它们在世界中的位置可能会变换（例如在它们直线坠落穿过基岩之时），我的梦仍然与他者的相邻。<BR>这是突破性的发现。作为一个比大多数人都更熟练的梦者，我能够用自己的梦去探查他人的梦，无视他们自以为拥有的隐私，增进对他们的了解。我应该继续探索这一能力，以便开发其全部潜能。<BR>然而，这一新的知识给我带来了新的恐惧。<BR>这种恐惧来自一个想法：或许有其他的梦者，比我更为强大，而我对梦境过度的入侵只会使我更容易受到损害。<BR>其他的梦者，他们甚至可能不是人类。<PAGE>关于人类的记忆能通过与哪些愚昧村民中的一员发起交易来唤起。他们迟早会为自己愚蠢的定价付出代价。<BR>这段记忆现在所能引发的梦境并不会带来很大影响。它使我能获知像我一样的其他人的位置，但只是提供除我之外的、姓名未知的某个人类所在的x或z坐标，前提是至少有一个人类正处于梦中。
```
“脆弱”难以理解，但是改成“更容易遭受损害”就好理解了，因为过度入侵就容易碰到那些强大的梦者，打不过，自然也就更容易受到损害了。

Determancer replied:
```suggestion
research.HUMANDREAMS.stage.1=我有理由相信，在其怪异的、非维度态的存在位面中，诸梦彼此相邻。每一个梦都邻接余众。在同一时刻，在任何时间。<BR>虽然它们在世界中的位置可能会变换（例如在它们直线坠落穿过基岩之时），我的梦仍然与他者的相邻。<BR>这是突破性的发现。作为一个比大多数人都更熟练的梦者，我能够用自己的梦去探查他人的梦，无视他们自以为拥有的隐私，增进对他们的了解。我应该继续探索这一能力，以便开发其全部潜能。<BR>然而，这一新的知识给我带来了新的恐惧。<BR>这种恐惧来自一个想法：或许有其他的梦者，比我更为强大，而我对梦境过度的入侵只会使我更容易受到伤害。<BR>其他的梦者，他们甚至可能不是人类。<PAGE>关于人类的记忆能通过与哪些愚昧村民中的一员发起交易来唤起。他们迟早会为自己愚蠢的定价付出代价。<BR>这段记忆现在所能引发的梦境并不会带来很大影响。它使我能获知像我一样的其他人的位置，但只是提供除我之外的、姓名未知的某个人类所在的x或z坐标，前提是至少有一个人类正处于梦中。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+research.VOIDDREAMS.title=虚空之梦
+research.VOIDDREAMS.stage.0=一段全新的记忆涌现在我的头脑之中。<BR>虚空。漆黑无底，神秘莫测，不可知晓。它的本质在我的脑海中燃烧并照亮了一切，之前的记忆没有一个与此相同。<BR>梦经行§l虚空§r。<BR>§l那§r即是它们真正脱离已知世界，前往无人能跟随到达的地方......至少目前无人能涉足彼处。<BR>我确信如此。<BR>我需要梦见§l虚空§r。<BR>我要看到它邃暗的虚无。<BR>如此一来我或许能亲身到达彼处。
+research.VOIDDREAMS.stage.1=这场梦境远没有我想象的那般震撼。<BR>虚空之梦似乎不会对我直接产生影响。<BR>然而，我在梦见虚空后（或是梦见虚空期间，如果我有意如此的话）再次入梦之时，我运梦的能力似乎有所提升。这一效果并不显著，并且仅能用于极少数的记忆，但它也因此具有意义。<BR>不过，我不应该低估这一记忆的重要性。<BR>我相信其真正的作用是放缓我下一次梦的沉降，尤其是梦在虚空中停留之时。如果这是真的，那么我的求知之途将取得重大突破。<BR>我§l需要§r知道是什么、或者是谁栖居于那片邃暗的虚无之中。我要知道梦是如何运作，它们前往何方，它们如何获得知识。我要知道何为真实，何为虚妄。<BR>我要知道真相。
```
|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
research.VOIDDREAMS.stage.1=这场梦境远没有我想象的那般震撼。<BR>虚空之梦似乎不会对我直接产生影响。<BR>然而，如果我在梦见虚空后（或是梦见虚空期间，如果我有意如此的话）再次入梦，我的运梦的能力似乎就会有所提升。这一效果并不显著，并且仅能用于极少数的记忆，但它也因此具有意义。<BR>不过，我不应该低估这一记忆的重要性。<BR>我相信其真正的作用是放缓我下一次梦的沉降，尤其是梦在虚空中停留之时。如果这是真的，那么我的求知之途将取得重大突破。<BR>我§l需要§r知道是什么、或者是谁栖居于那片邃暗的虚无之中。我要知道梦是如何运作，它们前往何方，它们如何获得知识。我要知道何为真实，何为虚妄。<BR>我要知道真相。
```
> were I to Dream again, after Dreaming of Void 

虚拟语气的书面表达法，should删掉，be动词或者助动词移至句首。

Determancer replied:
```suggestion
research.VOIDDREAMS.stage.1=这场梦境远没有我想象的那般震撼。<BR>虚空之梦似乎不会对我直接产生影响。<BR>然而，如果我在梦见虚空后（或是梦见虚空期间，如果我有意如此的话）再次入梦，我运梦的能力似乎就会有所提升。这一效果并不显著，并且仅能用于极少数的记忆，但它也因此具有意义。<BR>不过，我不应该低估这一记忆的重要性。<BR>我相信其真正的作用是放缓我下一次梦的沉降，尤其是梦在虚空中停留之时。如果这是真的，那么我的求知之途将取得重大突破。<BR>我§l需要§r知道是什么、或者是谁栖居于那片邃暗的虚无之中。我要知道梦是如何运作，它们前往何方，它们如何获得知识。我要知道何为真实，何为虚妄。<BR>我要知道真相。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.STRONGHOLD.stage.0=悚异之梦与末地有关，这没什么可惊奇的，毕竟它与虚空的关联如此密切。<BR>未被虚空之梦加强时，悚异之梦将告诉我一个要塞的确切坐标。
+
+research.DARKDREAMS.title=黑暗之梦
+research.DARKDREAMS.stage.0=在进入大海之前，我能感到沙子，它们在我的......我的梦境之下。颜色深沉、黏滑的暗沙，呈现出虚空的颜色。<BR>确实有报告提及潮湿、昏黑的沙滩，更进一步还有流言称这些“虚芜”（Voided）的生物群系定居着一小群深居简出的渔民。<BR>我想知道他们是否能为我的求知之途提供线索。无论如何，我现在的目标是梦见黑暗。这将是一段全新的、令人印象深刻的记忆。
```
|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
research.DARKDREAMS.stage.0=在进入大海之前，我能感到沙子，它们在我的......我的梦境之下。颜色深沉、黏滑的暗沙，呈现出虚空的颜色。<BR>确实有消息提及潮湿、昏黑的沙滩，更传闻称这些“虚芜”（Voided）的生物群系定居着一小群深居简出的渔民。<BR>我想知道他们是否能为我的求知之途提供线索。无论如何，我现在的目标是梦见黑暗。这将是一段全新的、令人印象深刻的记忆。
```
很奇妙，rumor确实可以不带贬义的翻译为“传闻”，在am3的文本里，出现了大量的“rumor”，我才第一次知道rumor原来在英语里可以不表现贬义。

Determancer replied:
rumor一般指的是谣言或半真半假的消息，而流言的义项包含“众人流传的话”，并不一定是贬义，因此此处流言并无不妥。
```suggestion
research.DARKDREAMS.stage.0=在进入大海之前，我能感到沙子，它们在我的......我的梦境之下。颜色深沉、黏滑的暗沙，呈现出虚空的颜色。<BR>确实有传闻提及潮湿、昏黑的沙滩，更有流言称这些“虚芜”（Voided）的生物群系定居着一小群深居简出的渔民。<BR>我想知道他们是否能为我的求知之途提供线索。无论如何，我现在的目标是梦见黑暗。这将是一段全新的、令人印象深刻的记忆。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.FISHINGHAMLET.title=旧渔村
+research.FISHINGHAMLET.stage.0=我需要真相。我受够了谎言。<BR>必然是有一道幻觉织就的帷幕遮挡了所有的人的心智。我所见的一切，皆为谎言。<BR>§l只有谎言。§r<BR>我必须揭开这道帷幕。我必须看到屏障后的彼端。<BR>无论旧渔村是解开终极奥秘的关键，或者仅仅是宏大谜题的一个碎片，我们终会知晓。
+research.FISHINGHAMLET.stage.1=这不可能...<BR>所有的工作，所有的研究...把我引到了这个地方？！<BR>到一群堕落者之中？！到一群排外的蠢货之中？！？<BR>不不不，我绝不能就此停止。我能揭开帷幕。我知道我能。我确信如此。我坚信不疑。<BR>我需要和灯塔看守谈谈。我希望比起他的居民同伴，他能表现得更加明智。
+research.FISHINGHAMLET.stage.2=看看大海...看看大海...
```
|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
research.FISHINGHAMLET.stage.2=望着大海...望着大海...
```

Determancer replied:
这里强调的是追随灯塔看守的指引去看大海，望着是静态的，看看则有动态的趋向。
例：“我一直想看看大海。”——我暂没看，希望去亲眼见到。
“我一直望着大海。”——我看见了，并一直注目。

TUsama replied:
其实主要是不知道 这句话发生在什么时候。

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.LHKeeper.text=与灯塔看守交谈。
+
+research.IDOL.title=共鸣
+research.IDOL.stage.0=有时候，我们会自以为神。万能的、无上的神，整个世界必须向我们俯首称臣。神，是造物主，能重塑整个世界。<BR>但我们就好像是易受哄骗的愚人。<BR>大海。<BR>无边，永恒，莫测。它真的在乎我们做了什么吗？当它让高大雄伟的建筑显得无比渺小，当它的年寿超越我们最为强劲坚固的巨型装置时，它真的会在乎吗？<BR>大海即是无限的象征，前所未有的接近无限，时刻提醒着我们意识到自己的§l微不足道§r。<BR>大海，它具有着令旧渔村居民无比神往的广远，是§l不可知的§r。我们可以费尽终生冀望研究它的每一个缝隙和角落，然而这一切都是徒劳，因为在它无边无际的领域以及它变化多端的混沌自然之中，居于其中的事物...或者居于其底的事物...往往会逃过我们的视线。除非它们主动来寻找我们。<BR>没错，大海一定就是关键。在它无边的、不可知的无限之中，§l必定§r隐藏着我们正 ...
```
|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

我查了查资料，我感觉这里的

> Could it really, when it dwarfs our greatest, grandest structures, when it outlives our strongest, toughest contraptions?

这两个when不是引导的和时间有关的从句，我查了一查，when居然还能引导原因状语从句，译为“既然，考虑到”，参考：https://zhidao.baidu.com/question/58722068.html
既然是这样的话，应该是：

```suggestion
research.IDOL.stage.0=有时候，我们会自以为神。万能的、无上的神，整个世界必须向我们俯首称臣。神，是造物主，能重塑整个世界。<BR>但我们就好像是易受哄骗的愚人。<BR>大海。<BR>无边，永恒，莫测。它真的在乎我们做了什么吗？它会因为它让高大雄伟的建筑显得无比渺小，会因为它的年寿超越我们最为强劲坚固的巨型装置，而对我们投来哪怕是一丝一毫的关注吗？<BR>大海即是无限的象征，前所未有的接近无限，时刻提醒着我们意识到自己的§l微不足道§r。<BR>大海，它具有着令旧渔村居民无比神往的广远，是§l不可知的§r。我们可以费尽终生冀望研究它的每一个缝隙和角落，然而这一切都是徒劳，因为在它无边无际的领域以及它变化多端的混沌自然之中，居于其中的事物...或者居于其底的事物...往往会逃过我们的视线。除非它们主动来寻找我们。<BR>没错，大海一定就是关键。在它无边的、不可知的无限之中，§l必定§r隐藏着我们正在寻找的一切。就如同某种门户，打开这扇门就能看到帷幕彼端。在大海无底的深渊中，我们一定要找到裂隙。<BR>水§l散发§r着悚异的气息，尤其是那个渔村中的古怪堕落者所崇拜的某种...某种...<BR>没错。某种...当然，他们绝非堕落者，因为他们远比§l我们§r更早地找到了解决方法。<BR>如果他们的神祇真的是帷幕彼端的展现形式，将真相从我们所有人眼前隐藏，如果他们对梦和海的狂热是出于与我们共同的天性...那么我们必须寻求共鸣。<BR>与那伟大长梦者共鸣。
```

Determancer replied:
```suggestion
research.IDOL.stage.0=有时候，我们会自以为神。万能的、无上的神，整个世界必须向我们俯首称臣。神，是造物主，能重塑整个世界。<BR>但我们就好像是易受哄骗的愚人。<BR>大海。<BR>无边，永恒，莫测。它真的在乎我们做了什么吗？既然它能让高大雄伟的建筑显得无比渺小，既然它的年寿超越我们最为强劲坚固的巨型装置，那么它真的会对我们投来哪怕是一丝一毫的关注吗？<BR>大海即是无限的象征，前所未有的接近无限，时刻提醒着我们意识到自己的§l微不足道§r。<BR>大海，它具有着令旧渔村居民无比神往的广远，是§l不可知的§r。我们可以费尽终生冀望研究它的每一个缝隙和角落，然而这一切都是徒劳，因为在它无边无际的领域以及它变化多端的混沌自然之中，居于其中的事物...或者居于其底的事物...往往会逃过我们的视线。除非它们主动来寻找我们。<BR>没错，大海一定就是关键。在它无边的、不可知的无限之中，§l必定§r隐藏着我们正在寻找的一切。就如同某种门户，打开这扇门就能看到帷幕彼端。在大海无底的深渊中，我们一定要找到裂隙。<BR>水§l散发§r着悚异的气息，尤其是那个渔村中的古怪堕落者所崇拜的某种...某种...<BR>没错。某种...当然，他们绝非堕落者，因为他们远比§l我们§r更早地找到了解决方法。<BR>如果他们的神祇真的是帷幕彼端的展现形式，将真相从我们所有人眼前隐藏，如果他们对梦和海的狂热是出于与我们共同的天性...那么我们必须寻求共鸣。<BR>与那伟大长梦者共鸣。
```

TUsama replied:
> Could it really, when it dwarfs our greatest, grandest structures, when it outlives our strongest, toughest contraptions?

你换成now that就好理解了。

> Could it really, now that it dwarfs our greatest, grandest structures, now that it outlives our strongest, toughest contraptions?

```suggestion
research.IDOL.stage.0=有时候，我们会自以为神。万能的、无上的神，整个世界必须向我们俯首称臣。神，是造物主，能重塑整个世界。<BR>但我们就好像是易受哄骗的愚人。<BR>大海。<BR>无边，永恒，莫测。它真的在乎我们做了什么吗？在它面前，高大雄伟的建筑显得无比渺小，而它的年寿又超越我们最为强劲坚固的巨型装置，但它会因此而对我们投来哪怕是一丝一毫的关注吗？<BR>大海即是无限的象征，前所未有的接近无限，时刻提醒着我们意识到自己的§l微不足道§r。<BR>大海，它具有着令旧渔村居民无比神往的广远，是§l不可知的§r。我们可以费尽终生冀望研究它的每一个缝隙和角落，然而这一切都是徒劳，因为在它无边无际的领域以及它变化多端的混沌自然之中，居于其中的事物...或者居于其底的事物...往往会逃过我们的视线。除非它们主动来寻找我们。<BR>没错，大海一定就是关键。在它无边的、不可知的无限之中，§l必定§r隐藏着我们正在寻找的一切。就如同某种门户，打开这扇门就能看到帷幕彼端。在大海无底的深渊中，我们一定要找到裂隙。<BR>水§l散发§r着悚异的气息，尤其是那个渔村中的古怪堕落者所崇拜的某种...某种...<BR>没错。某种...当然，他们绝非堕落者，因为他们远比§l我们§r更早地找到了解决方法。<BR>如果他们的神祇真的是帷幕彼端的展现形式，将真相从我们所有人眼前隐藏，如果他们对梦和海的狂热是出于与我们共同的天性...那么我们必须寻求共鸣。<BR>与那伟大长梦者共鸣。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+research.IDOL.title=共鸣
+research.IDOL.stage.0=有时候，我们会自以为神。万能的、无上的神，整个世界必须向我们俯首称臣。神，是造物主，能重塑整个世界。<BR>但我们就好像是易受哄骗的愚人。<BR>大海。<BR>无边，永恒，莫测。它真的在乎我们做了什么吗？当它让高大雄伟的建筑显得无比渺小，当它的年寿超越我们最为强劲坚固的巨型装置时，它真的会在乎吗？<BR>大海即是无限的象征，前所未有的接近无限，时刻提醒着我们意识到自己的§l微不足道§r。<BR>大海，它具有着令旧渔村居民无比神往的广远，是§l不可知的§r。我们可以费尽终生冀望研究它的每一个缝隙和角落，然而这一切都是徒劳，因为在它无边无际的领域以及它变化多端的混沌自然之中，居于其中的事物...或者居于其底的事物...往往会逃过我们的视线。除非它们主动来寻找我们。<BR>没错，大海一定就是关键。在它无边的、不可知的无限之中，§l必定§r隐藏着我们正 ...
+research.IDOL.stage.1=我谦卑地向你发誓献上我的生命，伟大长梦者。<BR>我否定过往的所有信仰，所有我曾深信不疑的神灵。<BR>我对你而言微不足道。大海，是你在主世界的化身，是你的象征之物。<BR>你对我而言是不可知的。遍布虚空的虚无，即是你高深莫测的本质。<BR>你所深藏的彼端万象皆存。那即是真相，被帷幕所藏。<BR>赐予我们知识，如此我们方能揭穿一切谎言。<BR>赐予我们力量，如此我们方能在迎接真相时幸存。<BR>赐予我们勇气...如此我们方能聆听你的神圣之音。
```
|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
research.IDOL.stage.1=我谦卑地向你发誓献上我的生命，伟大长梦者。<BR>我否定过往的所有信仰，所有我曾深信不疑的神灵。<BR>我对你而言微不足道。大海，是你在主世界的化身，是你的象征之物。<BR>你对我而言是不可知的。那遍布虚空的虚无，即是你高深莫测的本质。<BR>你所深藏的彼端万象皆存。那即是真相，被帷幕所藏。<BR>赐予我们知识，如此我们方能揭穿一切谎言。<BR>赐予我们力量，如此我们方能在迎接真相时幸存。<BR>赐予我们勇气...如此我们方能聆听你的神圣之音。
```

Determancer replied:
leave it.

TUsama reviewed (approved):
今早一个钟

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Clefal <1716976420@qq.com>

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.craftedlacrymatory.text=合成一个泪瓶。
+
+research.REVELATIONRING.title=启示之戒
+research.REVELATIONRING.stage.0=泣者和泣囊没有眼睛，然而他们所见远超常人。<BR>他们的眼泪非常珍贵，我应该用它们润泽一个戒指。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
research.REVELATIONRING.stage.0=泣者和泣囊没有眼睛，然而他们所见远超常人。<BR>他们的眼泪弥足珍贵，我应该用这些眼泪润泽一枚戒指。
```

TUsama commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+research.BONETIARA.title=骨冠
+research.BONETIARA.stage.0=如果以火攻火是千真万确的道理，那么我应该是第一个以恐惧对抗恐惧的人。<BR>没有哪种恐怖能够击垮我的意志。
+research.BONETIARA.stage.1=随着我知识的增长，我所看见的景象令我的脊骨瑟瑟发抖。<BR>但愿现在这种颤抖能到别人身上去。<BR>一顶骨冠现在可用作装饰我的头部。只要它的被动效果处于开启状态，我应该就能免疫恐惧和痴愚。我的意志牢不可破。<BR>至少目前如此。
```
|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
research.BONETIARA.stage.1=随着我知识的增长，我所看见的景象令我的脊骨瑟瑟发抖。<BR>但愿现在这种颤抖能到别人身上去。<BR>这一顶骨冠现在便是我头顶的装饰物了。只要它的被动效果处于开启状态，我应该就能免疫恐惧和痴愚。我的意志牢不可破。<BR>至少目前如此。
```

Determancer replied:
Will now相当于把now放在句首，直译为“现在……将……”，后半部分可发生变化。
“便是”听起来已经戴上，略失准确。
这里的adorn强调的应该是骨冠为饰品（Baubles），所以进行修改如下。
```suggestion
research.BONETIARA.stage.1=随着我知识的增长，我所看见的景象令我的脊骨瑟瑟发抖。<BR>但愿现在这种颤抖能到别人身上去。<BR>骨冠现在能作为我头部的饰品。只要其被动效果处于开启状态，我应该就能免疫恐惧和痴愚。我的意志牢不可破。<BR>至少目前如此。
```

TUsama reviewed (changes requested):
半个钟，这部分真的很慢

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Clefal <1716976420@qq.com>

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+tile.ghost_grass.name=幽灵草方块
+tile.arboreal_generator.name=造树之木
+tile.sapling_shrub.name=树种灌木
+tile.plant_terra.name=大地之植
```
不好听。

Determancer replied:
这个东西似乎没有实装，所以我不太清楚它是啥。暂时这样留着吧。

WuzgXY-GitHub replied:
。。。

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+interact.idol.notyet=无事发生。不知为什么，我感到如释重负。
+interact.idol.moreslugs=必须吃下更多蛞蝓才能与神像共鸣。
+interact.idol.power=新的深潜之力：长按深潜之力键（默认为'R'键）以选择深潜之力，点按键位则会施展选定的力量。
```
```suggestion
interact.idol.power=新的深潜之力：长按深潜之力键（默认为 'R' 键）以选择深潜之力，点按键位则会施展选定的力量。
```
空格？

Determancer replied:
游戏内测试表明，加上空格会出现迷之排版错乱。所以就这样吧 (x

WuzgXY-GitHub replied:
👀👀👀

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+interact.idol.moreslugs=必须吃下更多蛞蝓才能与神像共鸣。
+interact.idol.power=新的深潜之力：长按深潜之力键（默认为'R'键）以选择深潜之力，点按键位则会施展选定的力量。
+interact.idol.communion=我的皮肤在蠕动。共鸣已烙印于我身。
+interact.idol.slugs=我的脊椎嘎吱作响。那个声音绝不属于人类，我的意志在动摇。
```
```suggestion
interact.idol.slugs=我的脊骨嘎吱作响。那个声音绝不属于人类，我的意志在动摇。
```

Determancer replied:
这里不必完全统一。但我突然发现脊椎不好听。
```suggestion
interact.idol.slugs=我的脊柱嘎吱作响。那个声音绝不属于人类，我的意志为之动摇。
```

WuzgXY-GitHub replied:
不必？那行吧
后面这个不错

WuzgXY-GitHub reviewed (approved):
第二波
前两百行

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+gui.sleep_chamber.wake=醒来
+
+gui.tablet.inscription0=There is nothing in the Void.<BR>虚空无物。<BR>And yet everything comes from There.<BR>然亦万物之源。<BR>That black portal to Infinity.<BR>邃暗门后，可见无限。
+gui.tablet.inscription1=Knowledge is to be feared,<BR>知识宜惧，<BR>not sought.<BR>莫当索求。<BR>It may bring Power,<BR>虽携神力，<BR>but at what cost?<BR>代价何由？
```
```suggestion
gui.tablet.inscription1=Knowledge is to be feared,<BR>知识当惧，<BR>not sought.<BR>更莫索求。<BR>It may bring Power,<BR>虽携神力，<BR>but at what cost?<BR>代价何由？
```

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+gui.tablet.inscription0=There is nothing in the Void.<BR>虚空无物。<BR>And yet everything comes from There.<BR>然亦万物之源。<BR>That black portal to Infinity.<BR>邃暗门后，可见无限。
+gui.tablet.inscription1=Knowledge is to be feared,<BR>知识宜惧，<BR>not sought.<BR>莫当索求。<BR>It may bring Power,<BR>虽携神力，<BR>but at what cost?<BR>代价何由？
+gui.tablet.inscription2=Infinite Yous throughout infinite moments,<BR>亿万之汝，经瞬无穷，<BR>among infinite timelines<BR>宙列无数，汝之所由<BR>within infinite Infinities.<BR>众象莫尽，皆在无尽中。<BR>You are nothing. Do not forget that.<BR>汝唯苍穹一粟。慎勿忘之。
+gui.tablet.inscription3=The deepest Oceans touch the Void.<BR>至深之海，连触虚空。<BR>Water is unsafe, it is Chaos.<BR>其水非静，混沌其中。<BR>Yet from it comes Life.<BR>然众生于此初诞。
```
```suggestion
gui.tablet.inscription3=The deepest Oceans touch the Void.<BR>至深之海，连触虚空。<BR>Water is unsafe, it is Chaos.<BR>水渊无静，混沌其中。<BR>Yet from it comes Life.<BR>然众生于此初诞。
```

Determancer replied:
```suggestion
gui.tablet.inscription3=The deepest Oceans touch the Void.<BR>至深之海，连触虚空。<BR>Water is unsafe, it is Chaos.<BR>水渊无静，混沌其中。<BR>Yet from it comes Life.<BR>然众生于此初诞。
```

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+dagon.judgement.0=汝将入吾之列
+dagon.judgement.1=示吾汝有所值
+dagon.judgement.2=杀吾之远古守卫者三
```
```suggestion
dagon.judgement.2=杀吾守卫者三
```
牺牲『远古』二字以统一字数是否可行

Determancer replied:
不可行，变成小怪了

WuzgXY-GitHub reviewed (approved):
还剩 738 - 1081 行

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: WuzgXY <62000315+WuzgXY-GitHub@users.noreply.github.com>

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+research.crafteddream_focus_villagers.text=合成一台用于移动村民的梦之聚焦仪。
+
+research.SURGEON.title=外科医生
+research.SURGEON.stage.0=梦之网连接着所有窥探真相者。<BR>其整个社群存在之地都距我甚远，但其中的一些族类会为它们的盟友和庇护者效劳。<BR>我与大衮的结盟带来了新的联系，其中包括外科医生，它们会为寻找自己的求索者提供技能帮助。<BR>我必须合成一封传唤类型的信，以此让外科医生到我这来。<BR>然后我只需要对着地面使用信即可。我将完成此事并目睹外科医生的面容，随后再为他选择工作的具体内容。
```
```suggestion
research.SURGEON.stage.0=梦之网连接着所有窥探真相者。<BR>其整个社群存在之地都距我甚远，但其中的一些族类会为它们的盟友和庇护者效劳。<BR>我与大衮的结盟带来了新的联系，其中包括与外科医生的——它们会为寻找自己的求索者提供自己技能的帮助。<BR>我必须合成一封传唤类型的信，以让外科医生到我这来。<BR>然后我只需要对着地面使用信即可。我将完成此事并目睹外科医生的面貌，随后再为他选择工作的具体内容。
```
略冗长。

Determancer replied:
```suggestion
research.SURGEON.stage.0=梦之网连接着所有窥探真相者。<BR>其整个社群存在之地都距我甚远，但其中的一些族类会为它们的盟友和庇护者效劳。<BR>我与大衮的结盟使我能接触到新的族类，这其中包括外科医生，它们会为寻找自己的求索者提供技能帮助。<BR>我必须合成一封传唤类型的信，以此让外科医生到我这来。<BR>然后我只需要对着地面使用信即可。我将完成此事并目睹外科医生的面貌，随后再为他选择工作的具体内容。
```

WuzgXY-GitHub commented on translation/beyondtheveil/zh_cn.lang:
```diff
@@ -0,0 +1,1110 @@
+
+research.SURGEON.title=外科医生
+research.SURGEON.stage.0=梦之网连接着所有窥探真相者。<BR>其整个社群存在之地都距我甚远，但其中的一些族类会为它们的盟友和庇护者效劳。<BR>我与大衮的结盟带来了新的联系，其中包括外科医生，它们会为寻找自己的求索者提供技能帮助。<BR>我必须合成一封传唤类型的信，以此让外科医生到我这来。<BR>然后我只需要对着地面使用信即可。我将完成此事并目睹外科医生的面容，随后再为他选择工作的具体内容。
+research.SURGEON.stage.1=我很好奇外科医生为何能形成它们自己的社群。<BR>我过去从不知道外科医生并非职业。<BR>没错，它是一个名为外科医兽的种族。<BR>这一种族并不属于我所处的这个世界。<BR>这是一个被有意制造出来的种族吗？毕竟它们有一个共同的目标。<BR>......<BR>假如我同样是被*制造*出来的物种的一员，并非生于偶然，而是生于某种存在的意愿呢？我对此一无所知，毕竟我又怎会知道？<BR>外科医兽是个有用的侍从，不过称之为盟友可能更合适。<BR>我必须将他放置在沉梦水篮附近，在那里我想要他，她，或是它，随便哪个都行，去完成我安排的工作。<BR>然后我必须对他使用手术工具来指定我想要进行的操作。<BR>接下来，每当病人出现在水篮之中，外科医兽就会进行手术，然后将手术完成的病人放在设备顶部，并自己保留取出的心脏和脊骨。我接下来空手与之交互即可获取他所收 ...
```
```suggestion
research.SURGEON.stage.1=我曾经很好奇外科医生为何能形成它们自己的社群。<BR>我过去从不知道外科医生并非职业。<BR>没错，它是一个名为外科医兽的种族。<BR>这一种族并不属于我所处的这个世界。<BR>这是一个被有意制造出来的种族吗？毕竟它们有一个共同的目标。<BR>......<BR>假如我同样是被*制造*出来的物种的一员，并非生于偶然，而是生于某种存在的意愿呢？我对此一无所知。我又怎可能知道呢？<BR>一个外科医兽会是个有用的侍从，不过称之为盟友可能更合适。<BR>我必须将他放置在沉梦水篮附近，在那里我想要他，她，或是它，无所谓什么，去完成我安排的工作。<BR>然后我必须对他使用手术工具来指定我想要进行的操作。<BR>接下来，每当病人出现在水篮之中，外科医兽就会进行手术，然后将手术完成的病人放在设备顶部，并自己保留取出的心脏与脊骨。接下来我便可以空手与之交互并获取他收集到的所有材料。<BR>一个更为有用的技巧是在使用外科医生传唤信前将其与附近的箱子绑定。这样我就可以在手持传唤信时潜行与箱子交互了。<BR>我可以通过在潜行时右击空气来清除传唤信先前的绑定状态。<BR>由绑定的传唤信所召唤的外科医兽将会把所有手术的产物——包括病人和材料——放入与其绑定的箱子中，前提是箱子距它们足够近。<BR>由于物品形式的村民和泣者也能进入沉梦水篮，所以我可以高效地将外科医兽一个接一个的沿输送方向排布。<BR>外科医兽可由其主人使用棍棒来收起。
```
第一句是过去完成时。

Determancer replied:
```suggestion
research.SURGEON.stage.1=我曾经很好奇外科医生为何能形成他们自己的社群。<BR>这是因为我过去从不知道外科医生并非职业。<BR>没错，它是一个名为外科医兽的种族。<BR>这一种族并不属于我所处的这个世界。<BR>这是一个被有意制造出来的种族吗？毕竟它们有一个共同的目标。<BR>......<BR>假如我同样是被*制造*出来的物种的一员，并非生于偶然，而是生于某种存在的意愿呢？我对此一无所知，毕竟我又怎会知道？<BR>外科医兽是个有用的侍从，不过称之为盟友可能更合适。<BR>我必须将他放置在沉梦水篮附近，在那里我想要他，她，或是它，随便哪个都行，去完成我安排的工作。<BR>然后我必须对他使用手术工具来指定我想要进行的操作。<BR>接下来，每当病人出现在水篮之中，外科医兽就会进行手术，然后将手术完成的病人放在设备顶部，并自己保留取出的心脏和脊骨。我接下来空手与之交互即可获取他所收集到的所有材料。<BR>一个更为有用的技巧是在使用外科医生传唤信前将其与附近的箱子绑定。为此我可以在手持传唤信时潜行并与箱子交互。<BR>我可以通过在潜行时右击空气来清除传唤信先前的绑定状态。<BR>由绑定的传唤信所召唤的外科医兽将会把所有手术的产物，包括病人和材料，放入与其绑定的箱子中，前提是箱子距它们足够近。<BR>由于物品形式的村民和泣者也能进入沉梦水篮，所以我可以高效地将外科医兽一个接一个的沿输送方向排布。<BR>外科医兽可由其主人使用棍棒来捡起。
```

WuzgXY-GitHub replied:
```suggestion
research.SURGEON.stage.1=我曾经很好奇外科医生为何能形成他们自己的社群。<BR>这是因为我过去从不知道外科医生并非职业。<BR>没错，它是一个名为外科医兽的种族。<BR>这一种族并不属于我所处的这个世界。<BR>这是一个被有意制造出来的种族吗？毕竟它们有一个共同的目标。<BR>......<BR>假如我同样是被*制造*出来的物种的一员，并非生于偶然，而是生于某种存在的意愿的呢？我对此一无所知，而我又怎可能知道呢？<BR>外科医兽是有用的侍从，不过称之为盟友可能更合适。<BR>我必须将他放置在沉梦水篮附近，在那里我想要他，她，或是它，无所谓什么，去完成我安排的工作。<BR>然后我必须对他使用手术工具来指定我想要进行的操作。<BR>接下来，每当病人出现在水篮之中，外科医兽就会进行手术，然后将手术完成的病人放在设备顶部，并自己保留取出的心脏与脊骨。接下来我便可以空手与之交互并获取他收集到的所有材料。<BR>一个更为有用的技巧是在使用外科医生传唤信前将其与附近的箱子绑定。这样我就可以在手持传唤信时潜行与箱子交互了。<BR>我可以通过在潜行时右击空气来清除传唤信先前的绑定状态。<BR>由绑定的传唤信所召唤的外科医兽将会把所有手术的产物——包括病人和材料——放入与其绑定的箱子中，前提是箱子距它们足够近。<BR>由于物品形式的村民和泣者也能进入沉梦水篮，所以我可以高效地将外科医兽一个接一个的沿输送方向排布。<BR>外科医兽可由其主人使用棍棒来收起。
```

Determancer replied:
```suggestion
research.SURGEON.stage.1=我曾经很好奇外科医生为何能形成他们自己的社群。<BR>这是因为我过去从不知道外科医生并非职业。<BR>没错，它是一个名为外科医兽的种族。<BR>这一种族并不属于我所处的这个世界。<BR>这是一个被有意制造出来的种族吗？毕竟它们有一个共同的目标。<BR>......<BR>假如我同样是被*制造*出来的物种的一员，并非生于偶然，而是生于某种存在的意愿的呢？我对此一无所知，而我又怎可能知道呢？<BR>外科医兽是有用的侍从，不过称之为盟友可能更合适。<BR>我必须将他放置在沉梦水篮附近，在那里我想要他，她，或是它，无所谓什么，去完成我安排的工作。<BR>然后我必须对他使用手术工具来指定我想要进行的操作。<BR>接下来，每当病人出现在水篮之中，外科医兽就会进行手术，然后将手术完成的病人放在设备顶部，并自己保留取出的心脏和脊骨。接下来我便可以空手与之交互以获取他收集到的所有材料。<BR>一个更为有用的技巧是在使用外科医生传唤信前将其与附近的箱子绑定。这样我就可以在手持传唤信时潜行与箱子交互了。<BR>我可以通过在潜行时右击空气来清除传唤信先前的绑定状态。<BR>由绑定的传唤信所召唤的外科医兽将会把所有手术的产物——包括病人和材料——放入与其绑定的箱子中，前提是箱子距它们足够近。<BR>由于物品形式的村民和泣者也能进入沉梦水篮，所以我可以高效地将外科医兽一个接一个的沿输送方向排布。<BR>外科医兽可由其主人使用棍棒来拾起。
```

WuzgXY-GitHub committed:
Update translation/beyondtheveil/zh_cn.lang

Co-authored-by: Determancer <49300241+Determancer@users.noreply.github.com>

WuzgXY-GitHub commented:
懒得看了，直接过吧。

Determancer commented:
> 懒得看了，直接过吧。

啊这

Determancer committed:
Update translation/beyondtheveil/zh_cn.lang

WuzgXY-GitHub commented:
真就过了？

WuzgXY-GitHub commented:
To be honest, I don't want to translate mods like this any more. It seems that I've lost part of joy of playing when translating all the texts.

Determancer commented:
> To be honest, I don't want to translate mods like this any more. It seems that I've lost part of joy of playing when translating all the texts.

我是边玩边翻译的，所以不存在这个问题（逃）
